centurion.gemspec

```diff
@@ -34,7 +34,7 @@ Gem::Specification.new do |spec|
   spec.require_paths = ['lib']
 
   spec.add_dependency 'trollop'
-  spec.add_dependency 'excon', '~> 0.33'
+  spec.add_dependency 'excon', '~> 0.71.0'
```

Comment:
Small bump, LOL

Reply:
Good on them for maintaining compat though!